internal/proto/internal.proto

```diff
@@ -142,7 +142,7 @@ message RetrieveRequest {
   int64 limit = 11; // Optional
   bool ignoreGrowing = 12;
   bool is_count = 13;
-  int64 iteration_extension_reduce_rate = 14;
```

Comment:
this will break the compatibility, during rolling upgrading

Reply:
true, I overlooked this point

Reply:
changed